osu.Desktop/Security/CompatibilityModeChecker.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Linq;
+using Microsoft.Win32;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics;
+using osu.Game.Overlays;
+using osu.Game.Overlays.Notifications;
+
+namespace osu.Desktop.Security
+{
+    /// <summary>
+    /// Checks if the game is running with windows compatibility optimizations which could cause issues. Displays a warning notification if so.
+    /// </summary>
+    public partial class CompatibilityModeChecker : Drawable
```

Comment:
`ElevatedPrivilegesChecker.cs` seems to inherit `Component`, however `Component` seems to just be an alias for `Drawable`. Also, looking at the docs for `Component`, it looks to be implied that `Component` is deprecated, or will be deprecated. I decided to just use `Drawable` here as a result, but if it would be cleaner to use `Component`, it can be used

Reply:
Not quite, `Component` will eventually be changed so that `Drawable` inherits from `Component`. Use `Component` here. Only use `Drawable` when this class is meant to show something visually (this one doesn't, it signals the notification overlay to show a notification, not display something by itself).